### ISSUE TITLE
On Chromebook you can't download mcpe from google play store its not avalible 

### ISSUE THREAD

**Issue by Faeriechi** [ChristopherHX/mcpelauncher-manifest#246]
Apk Minecraft doesnt really work and when i used apk through linux it said "The specified file is not compatible with the launcher
Login to Google Play with an account owning Minecraft ( Playstore ) and let the launcher download compatible versions, including previous versions of Minecraft
Details:
This Launcher cannot load Minecraft (Not an aarch64 System) on this PC:
Not an aarch64 System

Valid Minecraft apk CPU architectures for this pc / launcher are x86_64" 

I can sign in with a account i have bought mc on and my email but it just says 
"To use the download feature, Minecraft: Bedrock Edition has to be purchased on the Google Play Store.
If you are trying to download a beta version, please make sure you are in the Minecraft beta program on Google Play. and then try again after a while (joining the program might take a while)."

**Comment by ChristopherHX** [comment]
> Apk Minecraft doesnt really work

Yeah, if you import a compatible apks it opens
- it will show you a license error if your license cannot be validated, quite too much piracy with this launcher in the past

I will remove / hide the use apk button later, you have to sign in.

> I can sign in with a account i have bought mc on and my email but it just says

Do you really own the game??
- You cannot buy Minecraft PE/bedrock if you only have a chromebook as device.
- Use an android emulator and buy the game there, you can delete the android emulator after buying and downloading the game there.
- Make shure you can download the game from another device or emulator.

You agreed to the tos prompt after sign in? if not it also fails, sign in again.

**Comment by Faeriechi** [comment]
Yes i own the game! I have bedrock on my phone and ipad and i have signed in about 4 times and restarted it just keeps on doing the same thing

**Comment by ChristopherHX** [comment]
> my phone

This phone runs googleplay android?
If it is an ipad, iphone, xbox, switch, playstation and win10 then you don't own it on googleplay.

**Comment by Faeriechi** [comment]
Yes it is google play it is a android

**Comment by Faeriechi** [comment]
My phone is connected to my chromebook so yeah
